external-crates/move/crates/move-analyzer/editors/code/README.md

```diff
@@ -1,76 +1,26 @@
-# move-analyzer
+# Move
 
-Provides language support for the Move programming language.
+Provides language support for the Move programming language. For information about Move visit the
+language [documentation](https://docs.sui.io/concepts/sui-move-concepts).
 
-Currently, this means a basic grammar and language configuration for Move (`.move`) that enables
-syntax highlighting, commenting/uncommenting, simple context-unaware completion suggestions while
-typing, and other basic language features in Move files.
-
-For information about Move visit [the Move repository](https://github.com/move-language/move).
-
-## How to Install
-
-The move-analyzer Visual Studio Code extension works via two components: the extension itself and
-the `move-analyzer` language server.
-
-### 1. Installing the `move-analyzer` language server<span id="Step1">
```

Comment:
I removed this entire part as this is not going to get published in VSCode Marketplace before I figure out how to install the move-analyzer binary automatically (without user's intervention), and I did not want to bother updating all this only to remove it in the next PR.